core/adapters/index.ts

```diff
@@ -26,11 +27,15 @@ export enum WarehouseType {
   BIGQUERY = "bigquery",
   POSTGRES = "postgres",
   REDSHIFT = "redshift",
-  SNOWFLAKE = "snowflake"
+  SNOWFLAKE = "snowflake",
+  SQLDATAWAREHOUSE = "sqldatawarehouse"
 }
 
 export function supportsCancel(warehouseType: WarehouseType) {
-  return warehouseType === WarehouseType.BIGQUERY;
+  if (warehouseType === WarehouseType.BIGQUERY || warehouseType === WarehouseType.SQLDATAWAREHOUSE)
```

Comment:
suggest pulling this out to an array called `supportsCancel` or something, and then doing

`return !!supportsCancel.find(supportsCancel)`